src/workflows/recipes/components/RecipeForm.js

```diff
@@ -27,6 +30,11 @@ export function cleanRecipeData(data) {
       error.data = { arguments: 'Invalid JSON.' };
       throw error;
     }
+  } else if (data.arguments.preferences) {
+    // XXX How is this going to work when it's PreferenceRollback??
```

Comment:
Not sure what to do about this comment.

Reply:
preference rollback only takes `rolloutSlug` as an argument, which is the slug of the preference rollout so it would never have `data.arguments.preferences` and we would not need to worry about this.

Reply:
we can probably just ditch this comment.